modules/ROOT/pages/Development/BeginnersGuide/SimpleMod/gameworldmodule.adoc

```diff
@@ -169,11 +171,63 @@ This mistake won't be apparent until you have some content to test later.
 If you'd like to learn more about root modules, you can read about it on the
 xref:Development/ModLoader/ModModules.adoc[Mod Modules] system page.
 
+== Compile and Save
+
+Now that we've created our first mod asset, we need to 'compile' and save it.
+
+The editor lets you know that your assets are unsaved through a variety of indicators:
+
+- In the Content Browser, an asterisk (*) will appear in the bottom-left corner of the asset's icon.
+
+image:BeginnersGuide/simpleMod/UnsavedAssetContentBrowser.png[Content Browser]
+
+- If you have the asset open, an asterisk will appear at the end of the asset's name in the tab in the top bar.
+
+image:BeginnersGuide/simpleMod/UnsavedAssetTab.png[Asset Tab]
+
+- If you have the asset editor panel open, the Compile button's icon will be differnet.
+
+image:BeginnersGuide/simpleMod/DirtyBlueprint.png[Dirty Blueprint - click this to compile it!]
+
+A common way to compile and Save is by clicking the 'compile' button in the top left of the asset editor panel.
+
+[TIP]
+====
+We suggest that you enable "Save on Compile: On Success Only" to skip having to also press the save button.
+
+image:BeginnersGuide/simpleMod/SaveOnCompileSuccess.gif[Enabling Save on Compile Success]
+====
+
+In Unreal Engine, compiling does not mean to package your mod for trying out in the game.
+Compiling is a data validation step that will check for errors before preparing the files to be packaged later.
+
+====
+Whenever you create any asset for your mod, or change existing ones, _*remember to compile and save it*_!
+====
+
+If you don't compile and save the file, next time you package and try out the mod, _*the changes you made will not be included with your mod*_ - this can be very confusing to troubleshoot!
```

Comment:
Is compiling actually required? Not sure I did compile myself
Also, could mention ctrl + shift + s to save all

Reply:
I think it may compile it for you at alpak time as long as it's saved, but compiling now will catch errors that would have just came up then, and with the auto-save-on-success, it's still one button click